.gitignore

```diff
@@ -1,5 +1,17 @@
-## Custom
-.idea/
```

Comment:
I have always found that not excluding all .idea files can result in some un-intended overhead on commits, which is not nice.

I also think that the codebase should not be specific to a certain IDE, and rely on maven as much as possible.

Reply:
I'd say that's a fair assessment. The idea behind selectivly excluding, rather than doing so in bulk, was based on [this KB article](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839). Returning to just excluding the whole of `.idea/` would be fine in this case.